lib/plugins/aws/invoke/index.js

```diff
@@ -28,7 +28,9 @@ class AwsInvoke {
     this.validate();
 
     // validate function exists in service
-    this.serverless.service.getFunction(this.options.function);
+    const functionObj = this.serverless.service.getFunction(this.options.function);
+    this.options.lambdaName = functionObj.name || `${this.serverless.service.service}-${
```

Comment:
it needs to be attached to the context so that the next `invoke` method can access it. Normal variables are out of reach in that case. We've been following that pattern in other plugins too.